### PR TITLE
DES-59 Fix full page refresh when authenticating

### PR DESCRIPTION
**Pull request by samtgarson**

### CODE REVIEW

File: cypress/fixtures/single-evidence-request-response.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "createdAt": "2020-11-30T15:34:12.299Z",
+  "resident": {
+    "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "name": "Namey McName",
+    "email": "frodo@bagend.com",
+    "phoneNumber": "+447123456780"
```
Comment (bogdan-zaharia) [inline cypress/fixtures/single-evidence-request-response.json:8]: should we have a real phone number?

File: src/pages/_app.test.tsx [hunk 2]
```diff
@@ -33,24 +33,55 @@ describe('CustomApp', () => {
...
         await CustomApp.getInitialProps(appContext);
 
         expect(serverSideRedirect).toHaveBeenCalledWith(
           res,
-          `/login?redirect=${ctx.pathname}`
+          `/login?redirect=%2Fpath`
         );
       });
 
+      it('returns `access denied` and redirects to login the page is private and rendered client side', async () => {
```
Comment (bogdan-zaharia) [inline src/pages/_app.test.tsx:74]: It feels like "when" is missing between "login" and "the"

File: cypress/fixtures/single-evidence-request-response.json (hunk 1 above)
Comment (samtgarson) [reply cypress/fixtures/single-evidence-request-response.json:8]: AFAIK it's a fake phone number (+447**12345678**)

**Comment by samtgarson** [comment]
@jhackett1 worth you giving this a quick scan, I consolidated the layouts slightly to DRY things up, please feedback if you have any 👍

**Review by bogdan-zaharia — APPROVED** [review]

**Pull request merged**